openwisp_monitoring/device/apps.py

```diff
@@ -26,10 +27,13 @@
 from .signals import device_metrics_received, health_status_changed
 from .utils import (
     get_device_cache_key,
+    handle_critical_check_change,
     manage_default_retention_policy,
     manage_short_retention_policy,
 )
 
+Check = load_model('check', 'Check')
```

Comment:
You can't load this here, the app will break because of this. Observe how it's done in `connect_device_signals`.